pkger/models.go

```diff
@@ -863,11 +869,17 @@ func (c chart) properties() influxdb.ViewProperties {
 func (c chart) validProperties() []failure {
 	var fails []failure
 
+	if c.Kind == chartKindMarkdown {
+		// at the time of writing, there's nothing to validate for markdown types
+		return fails
```

Comment:
nit: `return nil` here, and move the fails var down to closer to where it is being used,  above the validatorFns var